packages/p2p/src/utils/responsive.ts

```diff
@@ -0,0 +1,17 @@
+import { isMobile } from '@deriv/shared';
+
+/**
+ * Function to get desired text size based off responsive size
+ * @function getTextSize
+ * @returns {string} text size
+ */
+export const getTextSize = (mobile_size: string, desktop_size: string): string =>
```

Comment:
These 2 functions are the same why we are using 2 different methods 🤔

Reply:
true we could just make it one function, like getSize

Reply:
after checking, better to have it separated, cause ESlint keeps complaining if we make it as one function which returns either a number or string. Better we have two separate functions to handle two different types